python/tutorials/06-fused-attention.py

```diff
@@ -17,6 +17,9 @@
 import triton
 import triton.language as tl
 
+torch_dtype:tl.constexpr = torch.float16
+# torch_dtype:tl.constexpr = torch.float8_e5m2fnuz
```

Comment:
Is this the switch to choose between fp16 and fp8 as inputs?

Reply:
Yes, this is the switch. We can also use the type `torch.float8_34m3fnuz`.